pynicotine/gtkgui/ui/settings/uploads.ui

```diff
@@ -214,7 +214,7 @@
               <object class="GtkSpinButton" id="LimitSpeed">
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
-                <property name="tooltip_text">Kilobytes per second.</property>
+                <property name="tooltip_text">Kibibytes (2^10 bytes) per second.</property>
```

Comment:
Is there any use in including the bytes?

Reply:
I think it's a good idea.